scripts/src/main/resources/scripts/devon.properties

```diff
@@ -4,7 +4,7 @@
 # For details see https://github.com/devonfw/ide/blob/master/documentation/configuration.asciidoc
 #************************************************************************************************
 
-DEVON_IDE_TOOLS=(java mvn eclipse vscode node npm ng)
+DEVON_IDE_TOOLS=(java mvn eclipse vscode node npm ng ionic)
```

Comment:
As we already discussed, we should reduce the default tools setup by default rather than extending it.
If you need a tool, you can just use it via `devon «tool» «args»` and if not already installed, it will be installed on the fly. Further, if any project thinks they need `ionic` to be installed during the IDE setup, they can still put this into their settings GIT repo into `devon.properties`.
Many users have been complaining that the setup is already too fat and takes to long. Therefore we should go for more `opt-in` rahter than `opt-out`.
IMHO to take this further and make it into the release, I will better redo this PR without that change...